### PR TITLE
Store Bundles

### PR DESCRIPTION
**Pull request by Sledro**

### CODE REVIEW

File: node/celestia.go [hunk 1]
```diff
@@ -162,6 +162,9 @@ func (c *CelestiaClient) PublishBundle(blocks Bundle) (*CelestiaPointer, error)
 		c.logger.Warn("Failed to submit blob, retrying", "attempt", i+1, "fee", fee, "gas_limit", gasLimit, "gas_price", gasPrice, "error", err)
 
 		i++
+
+		// Delay between publishing bundles to Celestia to mitigate 'incorrect account sequence' errors
+		time.Sleep(30 * time.Second)
```
Comment (CryptoKass) [inline node/celestia.go:167]: This should probably be a config variables. `30`

**Review by CryptoKass — APPROVED** [review]
This is working  👍

In the future we could add a ttl as we don't need to hold on to stored blocks indefinitely, maybe using rocksdb instead of leveldb?

File: node/celestia.go (hunk 1 above)
Comment (CryptoKass) [reply node/celestia.go:167]: TODO later.

**Comment by Sledro** [comment]
> This is working 👍
> 
> In the future we could add a ttl as we don't need to hold on to stored blocks indefinitely, maybe using rocksdb instead of leveldb?

Yeah good idea @CryptoKass - will implement in the next version

**Pull request merged**